src/Components/AllArticles.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import getAllArticles from "../../utils/get-all-articles";
+
+function AllArticles() {
+  const [allArticles, setAllArticles] = useState([]);
+
+  useEffect(() => {
+    getAllArticles().then((articles) => {
+      setAllArticles(articles);
```

Comment:
It would be nice to have a loading state here that shows a user a request is ongoing